kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ConvertUtils.java

```diff
@@ -376,23 +378,23 @@ public static TaskInstance convertToTask(UserTaskInstanceDesc userTask) {
         return instance;
     }
     
-    public static TaskInstanceList convertToTaskInstanceListPO(Collection<UserTaskInstanceWithPotOwnerDesc> instances) {
+    public static TaskWithProcessDescriptionList convertToTaskInstanceListPO(Collection<UserTaskInstanceWithPotOwnerDesc> instances) {
```

Comment:
TaskInstanceList will be no longer used?

Reply:
it's used for all other types but not the PO customised one